File: RedSheet.DbEntities/Models/vArrivalAndOpeningTacticsRecord.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vArrivalAndOpeningTacticsRecords", Schema = "dbo")]
	[Serializable]
	public partial class vArrivalAndOpeningTacticsRecord  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int ArrivalAndOpeningTacticId { get; set; } 
 
	[Required] 
	[MaxLength(1000)]
	public string ArrivalAndOpeningTacticValue { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectEventTimelineId { get; set; } 

	public vArrivalAndOpeningTacticsRecord()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-arrival-and-opening-tactics-record.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vArrivalAndOpeningTacticsRecord {
    constructor(vArrivalAndOpeningTacticsRecord?: vArrivalAndOpeningTacticsRecord )  {
        let properties = [ "arrivalAndOpeningTacticId", "arrivalAndOpeningTacticValue", "projectEventTimelineId",];
        for (let property of properties)
            if (vArrivalAndOpeningTacticsRecord && vArrivalAndOpeningTacticsRecord[property])
                this[property] = vArrivalAndOpeningTacticsRecord[property];
    }
 
	arrivalAndOpeningTacticId : number =   0 ;
 
    @required()
    @maxLength(1000)
	arrivalAndOpeningTacticValue : string =   undefined;
 
    @range(1,2147483647)
	projectEventTimelineId : number =   undefined;


}
